deploy/etcd/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/coreos/etcd:v3.5.0
+
+RUN apt-get update && \
+    apt-get install -y iproute2 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* 
```

Comment:
Heh, perhaps extract that `etcd` binary and `COPY` it into a `FROM scratch` or `FROM alpine` container some day?

Reply:
I remember now, it's to introduce artificial latency, right?

In this case, perhaps a cleaner solution is to move it under some `tests/` or `integtests/` directory? Since `iproute2` is clearly not needed for real-life deployments.

Reply:
but we, probably, won't use docker compose for real-life deployment anyway?